### ISSUE TITLE
HASURA_GRAPHQL_CORS_DOMAIN doesn't support chrome-extension:// scheme

### ISSUE THREAD

**Issue by franciscolourenco** [hasura/graphql-engine#6345]
When connection through web sockets to hasura from a Chrome Extension with `HASURA_GRAPHQL_CORS_DOMAIN` enabled, hasura rejects the connection since Chrome includes a origin header like `chrome-extension://mdplmiioglkpgkgblfdilgebpppgdlm`.

Including `chrome-extension://mdplmiioglkpgkgblfdilgebpppgdlm` in the list of `HASURA_GRAPHQL_CORS_DOMAIN`, results in the following error at startup:

```
Fatal Error:- Environment variable HASURA_GRAPHQL_CORS_DOMAIN: invalid domain: 'chrome-extension://mdplmiioglkpgkgblfdilgebpppgdlm'. All domains should have scheme + (optional wildcard) host + (optional port)
```

**Comment by sdahlbac** [comment]
Same this happens with capacitor://localhost and serve://dist

**Comment by dminkovsky** [comment]
To flesh out the use-case mentioned by @sdahlbac:

The URL `capacitor://localhost` is used by [Capacitor](https://capacitorjs.com/), a Cordova-like framework for packaging webapps for iOS/Android using web views. On iOS, the webview it configures uses `capacitor://localhost` as the URL, which gets sent as the origin in CORS requests. So, it would be really nice if Hasura could be configured to accept this schema because otherwise the only option is to use `*`, which is not ideal from a security standpoint.

**Comment by dminkovsky** [comment]
Looks like this is a duplicate of #5818?

**Comment by tirumaraiselvan** [comment]
Yep, this is a duplicate of #5818 which has been fixed and released in latest v2.0 (alpha.11)

Closing this now, let us know if there are any issues.
